src/lib/menu/menu-trigger.ts

```diff
@@ -130,10 +129,10 @@ export class MdMenuTrigger implements AfterViewInit, OnDestroy {
   private _resetMenu(): void {
     this._setIsMenuOpen(false);
 
-    if (this._openedFromKeyboard) {
+    if (!this._openedByMouse) {
```

Comment:
Add comment here explaining _why_ we only want to set focus when it's opened via keyboard.